code/game/machinery/jukebox.dm

```diff
@@ -249,7 +249,7 @@ datum/track/New(var/title_name, var/audio)
 		new/datum/track("Don't Rush", 'sound/music/audioconsole/DontRush.ogg'),
 		new/datum/track("Phoron Will Make Us Rich", 'sound/music/audioconsole/PhoronWillMakeUsRich.ogg'),
 		new/datum/track("Amsterdam", 'sound/music/audioconsole/Amsterdam.ogg'),
-		new/datum/track("when", 'sound/music/audioconsole/When.ogg'),
+		new/datum/track("Whenever", 'sound/music/audioconsole/When.ogg'),
```

Comment:
```suggestion
		new/datum/track("when", 'sound/music/audioconsole/When.ogg'),
```

this is the actual name of the song don't change it thanks

Reply:
addressed.